Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue9326.xaml.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+/*
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Collections.ObjectModel;
+using System.Reactive;
+using ReactiveUI;
+*/
+
+#if UITEST
+using Xamarin.Forms.Core.UITests;
+using Xamarin.UITest;
+using NUnit.Framework;
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+    // To test this sample include <PackageReference Include="ReactiveUI" Version="11.2.3" /> in Xamarin.Forms.Controls
+    // and uncomment the ViewModel
+#if UITEST
+    [Category(UITestCategories.RefreshView)]
+#endif
+    [Preserve(AllMembers = true)]
+    [Issue(IssueTracker.Github, 9326, "[Bug] RefreshView should delay effects of IsEnabled while a refresh is happening and iOS is handling IsEnabled false incorrectly",
+		PlatformAffected.iOS)]
+	public partial class Issue9326 : TestContentPage
+	{
+		public Issue9326()
+		{
+#if APP
+			InitializeComponent();
+            //BindingContext = new Issue9326ViewModel();
+#endif
+        }
+
+		protected override void Init()
+		{
+
+		}
+    }
+
+	/*
+    public class Issue9326ViewModel : ReactiveObject
```

Comment:
Should this be comment out?

Reply:
Good question. The issue is using ReactiveUI refreshing and disabling the RefreshView at the same time. So, we need a reference to ReactiveUI package to test it. I include this comment: https://github.com/xamarin/Xamarin.Forms/blob/f309b8460dffef5b283b2a3370329ff05bd5e7a4/Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue9326.xaml.cs#L19
Should we include the reference an uncomment this lines?

Reply:
I don't think the UI test needs reactiveUI.. The behavior is just toggling CanExecute to false directly after executing so if you do the following order of commands


Execute => inside execute fire canexecutechanged which will set canexecute to false so the command gets disabled

At this point the RefreshView should stay visible